patches/components-content_settings-core-common-content_settings.h.patch

```diff
@@ -1,8 +1,12 @@
 diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
-index 824c42112ec33d18796cfe50bbec8cc4b62de660..2a54e9451e2a947a676495e4df3ff6528dbb0094 100644
+index 1203cc6ada248375bce7cb37054bcf0cd9957ea0..c7a3917e0bc4cc54cb1203f43ab1f4279cdda988 100644
 --- a/components/content_settings/core/common/content_settings.h
 +++ b/components/content_settings/core/common/content_settings.h
-@@ -76,6 +76,8 @@ struct RendererContentSettingRules {
+@@ -72,9 +72,12 @@ struct RendererContentSettingRules {
+   ~RendererContentSettingRules();
+   ContentSettingsForOneType image_rules;
+   ContentSettingsForOneType script_rules;
++  ContentSettingsForOneType autoplay_rules;
```

Comment:
see https://github.com/brave/brave-core/pull/4374/files#r369669453 for header patches